cloud-chatops/src/events.py

```diff
@@ -0,0 +1,64 @@
+"""
+This module contains events to run in main.py.
+"""
+
+import asyncio
+import schedule
+from features.pr_reminder import PostPRsToSlack
+from features.post_to_dms import PostToDMs
+from read_data import get_config
+
+PULL_REQUESTS_CHANNEL = "C03RT2F6WHZ"
+
+
+def run_global_reminder(channel) -> None:
+    """This event sends a message to the specified channel with all open PRs."""
+    PostPRsToSlack().run(channel=channel)
+
+
+def run_personal_reminder() -> None:
+    """This event sends a message to each user in the user map with their open PRs."""
+    users = list(get_config("user-map").values())
+    PostToDMs().run(users=users, post_all=False)
+
+
+async def schedule_jobs() -> None:
+    """
+    This function schedules tasks for the async loop to run.
```

Comment:
Nit: we should add that this schedules tasks for production use-cases and contains the hard-coded days and times